src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.utils.tsx

```diff
@@ -1,15 +1,19 @@
 import { Controller } from 'react-hook-form';
 import { Radio, RadioGroup, FormControlLabelProps } from '@mui/material';
 
-import { Table, UiType } from 'components';
-import { AppletImage } from 'features/Applets/Table/AppletImage/AppletImage';
-import { RadioGroupController } from 'components/FormComponents';
-import { WorkspaceImage, WorkspaceUiType } from 'features/SwitchWorkspace';
-import { Workspace } from 'features/SwitchWorkspace/SwitchWorkspace.types';
-import { StyledFlexTopCenter, StyledLabelLarge, StyledBodyLarge } from 'styles/styledComponents';
-import { variables } from 'styles/variables';
-import { HeadCell } from 'types/table';
-import theme from 'styles/theme';
+import { Table, UiType } from 'shared/components';
+import { AppletImage } from 'modules/Dashboard/features/Applets/Table/AppletImage';
```

Comment:
Can we move this component to shared please?
We use it in dashboard and library at the moment